build.py

```diff
@@ -36,6 +36,12 @@ def create_parser():
     parser.add_argument("-j", "--jobs", type=int, default=1,
                         help="How many parallel jobs to run (if supported). "
                              "Integer, default 1.")
+    parser.add_argument(
+        "-o",
+        "--output-dir",
+        default="build",  # synchronise with render.yaml -> deploy step
+        help="Output directory, relative to root. Default 'build'.",
+    )
```

Comment:
Kinda out of scope for this PR, but I'm not entirely sure why we're going to all the trouble of manually duplicating the arguments from Sphinx's own build command in the first place, for what appears to be the tiny convenience of typing `python build.py` instead of `python -m sphinx . build` for the small subset of users both building locally and not using the Makefile (assuming they remember the bespoke pattern and args to begin with).

The only other thing this file does AFAIK is copy PEP 0 to the root, but the Sphinx extension just [does that anyway](https://github.com/python/peps/blob/b1ad37039fe391ce93df705c9ac469be23cd1e00/pep_sphinx_extensions/__init__.py#L63) if not run from `build.py` by [calling the very same function from this file](https://github.com/python/peps/blob/b1ad37039fe391ce93df705c9ac469be23cd1e00/pep_sphinx_extensions/__init__.py#L67), so it would seem that function should simply be moved into the extension (which would make the latter fully self-contained).